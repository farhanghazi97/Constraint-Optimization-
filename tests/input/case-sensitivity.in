# Test for case sensitivity for meeting names
meeting, meeting-one
meeting, MEETING-ONE
domain, meeting-one, wed, hard
domain, meeting-one, 4pm, hard
domain, MEETING-ONE, tue, hard
domain, MEETING-ONE, 2pm, hard
