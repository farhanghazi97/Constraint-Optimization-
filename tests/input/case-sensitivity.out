meeting-one:wed 4pm
MEETING-ONE:tue 2pm
cost:0
